tox.ini

```diff
@@ -39,3 +38,8 @@ deps =
 commands =
     coverage run aggregate_tests.py
     coverage report -m
+
+[testenv:lint]
```

Comment:
We also need to [adopt this change in .travis.yml](https://github.com/in-toto/in-toto/blob/c775e3b260a50ec5fb673a9b725e2d2eacedfb6b/.travis.yml#L20-L21). And should we maybe move bandit to the lint env too?

Reply:
Ugh, I always forget to touch `.travis.yml` – thanks!